crates/storage/db/src/tables/mod.rs

```diff
@@ -13,6 +13,7 @@
 //! TODO(onbjerg): Find appropriate format for this...
 
 pub mod codecs;
+mod extensions;
```

Comment:
hmmm I think it's debatable. Why not have it inside the provider? cc @mattsse

Reply:
hm, yeah why not indeed it makes way more sense tbh